src/vmm/src/vcpu/mod.rs

```diff
@@ -94,15 +99,36 @@ impl Vcpu {
     }
 
     /// Configure regs.
-    pub fn configure_regs(&self, kernel_load: GuestAddress) -> Result<()> {
+    pub fn configure_regs(
```

Comment:
Is it possible to test this one?

Reply:
It's covered by the integration tests, are you thinking unit tests for each function here?

Reply:
I was thinking of something similar to [this](https://github.com/firecracker-microvm/firecracker/blob/master/src/arch/src/x86_64/regs.rs#L263) (and insisting on the bzImage scenario).